packages/f/frozen-bubble/files/frozen-bubble.desktop

```diff
@@ -4,5 +4,6 @@ Comment=Frozen Bubble arcade game
 Exec=frozen-bubble
 Icon=frozen-bubble
 Terminal=false
+StartupWMClass=perl
```

Comment:
Out of curiosity, why is the startup class 'perl'?

Reply:
![Screenshot from 2024-01-26 05-15-51](https://github.com/getsolus/packages/assets/101771435/f40cc138-665f-42c6-8bfe-258ae3d44591)

I don't like it, but it works. Is there any tool on Budgie like this(Looking Glass)? to check wmclass?

Reply:
You should be able to run `xprop WM_CLASS` in a terminal, and then click on the window you want to inspect.

Reply:
it is the same after all in Plasma and Budgie. I hope this is a non-issue